### ISSUE TITLE
update the quasar fee denom 

### ISSUE THREAD

**Issue by akure** [cosmology-tech/chain-registry#77]
Hi, I have updated the missing fee denom in the quasar testnet and mainnet in the cosmos/chain-registry

Here is the merged  PR - 
https://github.com/cosmos/chain-registry/pull/4052 

Mainnet -
```

{
        "denom": "uqsr",
        "fixed_min_gas_price": 0.01,
        "low_gas_price": 0.01,
        "average_gas_price": 0.025,
        "high_gas_price": 0.03
}
```
Testnet -

```

"fees": {
    "fee_tokens": [
      {
        "denom": "uqsr",
        "fixed_min_gas_price": 0,
        "low_gas_price": 0.0025,
        "average_gas_price": 0.025,
        "high_gas_price": 0.04
      }
    ]
  },
```

**Comment by akure** [comment]
@Zetazzz  Could you please check and update the cosmology npm chain registry.

**Comment by akure** [comment]
Hey @Zetazzz  Let me know if there is any query for this?

**Comment by Zetazzz** [comment]
HI, will publish soon, thx!

**Comment by Zetazzz** [comment]
published chain-registry@1.29.0
